src/logic/map_objects/tribes/tribe_basic_info.h

```diff
@@ -50,15 +50,17 @@ struct TribeBasicInfo {
 	explicit TribeBasicInfo(std::unique_ptr<LuaTable> table);
 
 	/// Internal name to reference this tribe
-	std::string name;
+	const std::string name;
+	/// Filepath of the tribe's icon
+	const std::string icon;
+	/// Filepath of the tribe's loading script
+	const std::string script;
 	/// Who designed this tribe
 	std::string author;
```

Comment:
Just by the way. Is this actually used somewhere? Are there plans ever to use it anywhere?

Reply:
Not right now, but it may become interesting when we add more modding support. So, let's not remove it just yet.